sysutils/snapraid/Portfile

```
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        amadvance snapraid 12.3 v
+
+name                snapraid
+version             12.3
+categories          sysutils
+
+license             GPL-3
+maintainers         {anquieta @anquieta}
+description         Backup program for disk arrays
+long_description    SnapRAID stores parity information of your data and \
+                    it recovers from up to six disk failures
+homepage            http://www.snapraid.it/
+
+github.tarball_from downloads
```

`github.tarball_from downloads` is the default, and is actually less preferred than `releases` foremost, followed by `archive`. `archive` is Github's newer generation links for archives, and `releases` uses the release archives directly uploaded by the project owner.  Since `releases` tarballs don't follow the standard Github naming, you may need to set the name using `distname`:

```suggestion
github.tarball_from release
distname            ${name}-${version}
```

You can give `releases` a try, but go back to `archive` if it's not working.